backend/clickhouse/log_row.go

```diff
@@ -46,6 +29,12 @@ type LogRow struct {
 }
 
 func NewLogRow(timestamp time.Time, projectID uint32, opts ...LogRowOption) *LogRow {
+	// if the timestamp is zero, set time
+	// TODO(et) - should we move this up to extractFields?
```

Comment:
should this be a new ticket or can this be handled in this PR?

Reply:
Tracked here for now #6116.

We can dig into this in our next pairing.